framework/src/modules/nft/events/all_nfts_from_chain_suported.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import { BaseEvent, EventQueuer } from '../../base_event';
+import { LENGTH_CHAIN_ID } from '../constants';
+
+export interface AllNFTsFromChainSupportedEventData {
+	chainID: Buffer;
+}
+
+export const supportAllNFTsFromChainEventDataSchema = {
+	$id: '/nft/events/supportAllNFTsFromChain',
+	type: 'object',
+	required: ['chainID'],
+	properties: {
+		chainID: {
+			dataType: 'bytes',
+			minLength: LENGTH_CHAIN_ID,
```

Comment:
There is a same constant with name `CHAIN_ID_LENGTH`. probably better to keep same name for same constants across modules. @shuse2 If the LIPs have diff names do we also need to inform the author ?

Reply:
yea let's just ping them 👍